data/content/agent_attributes.yaml

```
@@ -154,6 +154,16 @@ attributes:
   required: false
   desc: |
         The DogStatsD instance to send metrics to using UDP.
+- name: metrics-datadog-distributions
+  env_var: BUILDKITE_METRICS_DATADOG_DISTRIBUTIONS
+  default_value: "false"
+  requires: false
+  desc: |
+      Use Datadog Distributions for Timing metrics. By default, we submit metrics via standard gauges. This will unfortunately cause issues if you have multiple agents running,
+      as components like buildkite.jobs.duration.success.median becomes a roll up of the median execution time _from each agent_. As each agent is likely only running a single job
+      at a time, this causes all time related histogram metrics (avg, max, median, 95percentile) from a given agent to be the same value.
+      Distributions, however, aggregate the metrics server side at Datadog, thus the median and 95percentile is calculated given the set of builds executing across all agents.
+      Learn more about [Datadog Distributions](https://docs.datadoghq.com/metrics/types/?tab=distribution#metric-types)
```

As you mentioned, I think this is a bit too much information. I see that most values match the descriptions given in the [command references](https://buildkite.com/docs/agent/v3/cli-start#metrics-datadog-distributions), but adding a sentence for why someone would use this is helpful. Thinking:

```suggestion
      Use [Datadog Distributions](https://docs.datadoghq.com/metrics/types/?tab=distribution#metric-types) for timing metrics. This is helpful when running multiple agents as Datadog aggregates metrics server side.
```

What do you think?

---

I think the disclaimer is still useful to have. Perhaps, it's needed elsewhere too, but we can phrase it here like
```suggestion
      Use [Datadog Distributions](https://docs.datadoghq.com/metrics/types/?tab=distribution#metric-types) for timing metrics. This advisable when running multiple agents. Otherwise, metrics from multiple agents may be rolled up and appear to have the same value.
```